index.html

```diff
@@ -8,62 +8,194 @@
-			      <img src="assets/images/linkedin.png" alt="" class="icon">
-				</a>
-			  </li>
+          <div class="projects">
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
```

Comment:
Kindly use a `hx` a header tag, this represent the heading of an individual card. This is for correct semantics and for screen readers to understand your code.

Reply:
done @Nyame-Wolf